scripts/reload.sh

```
@@ -577,6 +586,24 @@ if should_skip_ghostty_cli_helper_zig_build; then
+  RESCUE_XCCONFIG_FILE="${TMPDIR:-/tmp}/cmux-reload-${TAG_SLUG}-rescue.xcconfig"
+  {
+    if [[ -n "${XCODE_XCCONFIG_FILE:-}" ]]; then
+      printf '#include "%s"\n' "$XCODE_XCCONFIG_FILE"
+    fi
+    printf 'SWIFT_ENABLE_BATCH_MODE = NO\n'
+    printf 'SWIFT_COMPILATION_MODE = singlefile\n'
+    printf 'COMPILER_INDEX_STORE_ENABLE = NO\n'
+    printf 'SWIFT_INDEX_STORE_ENABLE = NO\n'
+    printf 'GCC_GENERATE_DEBUGGING_SYMBOLS = NO\n'
+    printf 'DEBUG_INFORMATION_FORMAT = dwarf\n'
+    printf 'OTHER_SWIFT_FLAGS = $(inherited) -D CMUX_RESCUE_BUILD\n'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)